src/renderer/views/Popular/Popular.js

```diff
@@ -0,0 +1,108 @@
+import { defineComponent } from 'vue'
```

Comment:
This file no longer exists in dev and was migrated to Popular.vue (composition api)

Reply:
Good catch, I missed this on resolving the merge conflict. Updated now